Prox/Prox/Data/PlacesController.swift

```diff
@@ -44,6 +44,8 @@ class PlacesProvider {
 
     fileprivate var firstFetch = true
 
+    fileprivate let placesLock = NSLock()
```

Comment:
Move this declaration next to `var places` and add a comment to make it clear it should be used for all `places` accesses.